### PR TITLE
Engine switchover+Aegis 2.0 design.

### PR DESCRIPTION
**Pull request by Flaksim**
DO NOT MERGE untill Cael can confirm his new engines work, or all vessels will be immobile.

  - maptweak: "Switched over all ships to the new engine system."
  - rscadd: "Finishes up the UNSC Aegis 2.0 design."
  - rscadd: "Simplemob tweak."
  - rscadd: "Icon change in closet.dmi, added extra state."
  - rscadd: "Adds an ONI ID Badge type."
  - rscadd: "Adds 4 new whitelisted jobs for ONI crew and ODST troopers for UNSC Aegis."
  - rscdel: "Removed superfluous backup files."

### CODE REVIEW

File: code/modules/halo/overmap/fusion_thruster.dm [hunk 1]
```diff
@@ -38,11 +38,12 @@
 
 
 /obj/machinery/fusion_thruster
-	name = "repulsor engine"
+	name = "Fusion Thruster"
```
Comment (caelaislinn) [inline code/modules/halo/overmap/fusion_thruster.dm]: These two thruster fixes are in #830 and will conflict once mine is merged

File: maps/first_contact/maps/Covenant Corvette/overmap.dm [hunk 2]
```diff
@@ -6,6 +6,7 @@
 	icon = 'maps/first_contact/maps/Covenant Corvette/Corvette2.dmi'
 	icon_state = "ship"
 	fore_dir = WEST
+	vessel_mass = 3
```
Comment (caelaislinn) [inline maps/first_contact/maps/Covenant Corvette/overmap.dm:9]: I haven't tested mass in single digits. The thrusters produce 1 thrust each so mass this low may have unpredictably fast acceleration.

**Review by caelaislinn — CHANGES_REQUESTED** [review]
You'll need to revert the fixes to fusion_thruster

File: code/modules/halo/overmap/fusion_thruster.dm (hunk 1 above)
Comment (Flaksim) [reply code/modules/halo/overmap/fusion_thruster.dm]: Done.

File: maps/first_contact/maps/Covenant Corvette/overmap.dm (hunk 2 above)
Comment (Flaksim) [reply maps/first_contact/maps/Covenant Corvette/overmap.dm:9]: I tested this extensively, these values used generate ship speeds between 3 and 5.6 depending on the ship and the value I set.

**Comment by Flaksim** [comment]
This can be merged AFTER pr 830 is merged.

File: code/modules/halo/overmap/fusion_thruster.dm [hunk 3]
```diff
@@ -43,6 +43,7 @@
 	icon = 'fusion_thruster.dmi'
 	icon_state = "nozzle0"
 	use_power = 1
+	anchored = 1
```
Comment (caelaislinn) [inline code/modules/halo/overmap/fusion_thruster.dm]: Oops I meant all the file changes. This has to go as well

File: code/modules/halo/overmap/fusion_thruster.dm [hunk 4]
```diff
@@ -53,7 +54,7 @@
 	var/on = 1
 	var/datum/ship_engine/fusion/controller
 	var/obj/item/fusion_fuel/held_fuel
-	var/fuel_consumption_rate = 10
+	var/fuel_consumption_rate = 0.5
```
Comment (caelaislinn) [inline code/modules/halo/overmap/fusion_thruster.dm]: And this

**Review by caelaislinn — CHANGES_REQUESTED** [review]
The whole file needs to be reverted

File: code/modules/halo/overmap/fusion_thruster.dm (hunk 4 above)
Comment (Flaksim) [reply code/modules/halo/overmap/fusion_thruster.dm]: Done.

File: code/modules/halo/overmap/fusion_thruster.dm (hunk 3 above)
Comment (Flaksim) [reply code/modules/halo/overmap/fusion_thruster.dm]: Done.

**Review by caelaislinn — APPROVED** [review]

**Pull request merged**